<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mind UI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>brain.png</normaloff>brain.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Training</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_training_generation">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_training_filename">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Filename</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_training_filename">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Brain.save</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_training_generation">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Generation</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_training_infinite">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>88</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Infinite</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_training_go">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Go !</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_score">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>180</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_gencount">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>70</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>6</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_training_stop">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>241</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Stoooop !</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_training_gencount">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>50</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Generation Count</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_training_score">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>160</y>
        <width>61</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Score</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_max_score">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>300</y>
        <width>231</width>
        <height>61</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_training_maxscore">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>280</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Max Score</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_training_save">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Save !!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Playing</string>
     </attribute>
     <widget class="QLabel" name="label_playing_file">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Brain File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_playing_play">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Go Play</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_playing_filename">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Brain.save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_playing_playlast">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>271</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Play Last Brain</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
